PowerEditor/src/tools/NppModernShell/Packaging/AppxManifest.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:uap2="http://schemas.microsoft.com/appx/manifest/uap/windows10/2"
+  xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  xmlns:desktop="http://schemas.microsoft.com/appx/manifest/desktop/windows10"
+  xmlns:desktop4="http://schemas.microsoft.com/appx/manifest/desktop/windows10/4"
+  xmlns:desktop5="http://schemas.microsoft.com/appx/manifest/desktop/windows10/5"
+  xmlns:uap10="http://schemas.microsoft.com/appx/manifest/uap/windows10/10"
+  xmlns:com="http://schemas.microsoft.com/appx/manifest/com/windows10"
+  IgnorableNamespaces="uap uap2 uap3 rescap desktop desktop4 desktop5 uap10 com">
+  <Identity Name="NotepadPlusPlus" ProcessorArchitecture="neutral" Publisher="CN=&quot;Notepad++&quot;, O=&quot;Notepad++&quot;, L=Saint Cloud, S=Ile-de-France, C=FR" Version="1.0.0.0" />
```

Comment:
Are Location, Provence and Country needed here? 
With the renew of certificate every 3 years, these data could be changed, especially the locvation.

Reply:
I think it is, you can test it easily, just remove it, recreate the msix package and then try to sign it.
If signtool will sign it without errors, you should be good to go. (though you might need to reinstall the package to test that it works)

Reply:
@GurliGebis 
Sure, I will test it. I will keep you posted if it works.

Reply:
👍

Reply:
@GurliGebis 
OK, then let's keep only publisher/CN then.

Reply:
@donho okay, just to verify, it allowed you to sign it with just the CN part of it?

Reply:
@GurliGebis 
I have not checked it yet - I am working on packaging NSIS script, and signing problem (Windows 11 SDK has brought the new SignTool.exe which has the different behaviour from the previous one).
I will check later after v8.5.1 being released, so let's keep it as it is.